go.mod

```diff
@@ -1,3 +1,5 @@
+replace github.com/insomniacslk/dhcp => gitlab.com/Vitrifi/insomniacslk-dhcp v0.0.1-cne
```

Comment:
This doesn't look right

Reply:
Hello. Thank you for comment. It's what we used internally to get it working quick and dirty. It'm in a process  of cleaning that up. 

As is it may not meet open source project standards yet.

Reply:
By the way, that Pull Request was supposed to be inside our fork, but it seems I've done it wrongly.